### ISSUE TITLE
Applications should be before Workspaces

### ISSUE THREAD

**Issue by spirulin** [pop-os/gnome-shell-extension-pop-cosmic#235]
So Applications is like Start in Windows. And it's used all the time. So it's weird and takes more concentration to hit Applications. Usually, your muscle memory especially after Windows is searching in the end for a "start menu/application menu".

So please consider before releasing 21.10 to change their places or at least add an option user to change them. For example, if you disable both, then if you Enable first Application to go to position one and then enable Workspaces to go on two.

Thanks for you work.

**Comment by senntore** [comment]
> For example, if you disable both, then if you Enable first Application to go to position one and then enable Workspaces to go on two.

I agree with this, though for now you can use this extension [Top Bar Organizer](https://extensions.gnome.org/extension/4356/top-bar-organizer/) to position them however you want.
